src/components/board/SharingTabSidebard.vue

```diff
@@ -33,24 +33,25 @@
 				<label for="checkbox">{{ t('deck', 'Manage') }}</label>
 
 				<button v-tooltip="t('deck', 'Delete')" class="icon-delete" @click="clickDeleteAcl(acl)" />
-
 			</li>
 		</ul>
-		<CollaborationView />
+
+		<collection-list v-if="board.id" :id="&quot;board.id&quot;" :name="board.title"
+			type="deck" />
 	</div>
 </template>
 
 <script>
 import { Avatar, Multiselect } from 'nextcloud-vue'
-import CollaborationView from '../CollaborationView'
+import { CollectionList } from 'nextcloud-vue-collections'
```

Comment:
Let's delete the CollaborationView component then, as it is not needed any longer.